gnd/src/main/java/com/google/android/gnd/ui/home/mapcontainer/FeatureRepositionViewModel.java

```diff
@@ -21,27 +21,26 @@
 import com.google.android.gnd.rx.annotations.Hot;
 import com.google.android.gnd.ui.common.AbstractViewModel;
 import com.google.android.gnd.ui.common.SharedViewModel;
-import com.google.android.gnd.ui.map.MapProvider;
 import io.reactivex.Observable;
 import io.reactivex.subjects.PublishSubject;
 import io.reactivex.subjects.Subject;
+import java.util.Objects;
+import javax.annotation.Nullable;
 import javax.inject.Inject;
 
 @SharedViewModel
 public class FeatureRepositionViewModel extends AbstractViewModel {
 
-  private final MapProvider mapProvider;
-
   @Hot private final Subject<Point> confirmButtonClicks = PublishSubject.create();
   @Hot private final Subject<Nil> cancelButtonClicks = PublishSubject.create();
+  @Nullable private Point lastKnownCameraTarget;
```

Comment:
You can probably drop "lastKnown", since all states carry their last known values.